.github/workflows/node-ci.yml

```diff
@@ -140,11 +146,43 @@ jobs:
             - name: Jest with coverage
               run: yarn coverage
 
+            - name: Upload Artifact
+              uses: actions/upload-artifact@v2
+              with:
+                  name: jest-coverage
+                  path: ./coverage/coverage-final.json
+
+    upload_coverage:
+        name: Upload Coverage
+        runs-on: ubuntu-latest
+        needs: [cypress, coverage]
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Use Node.js ${{ matrix.node-version }} & Install & cache node_modules
+              uses: Khan/actions@shared-node-cache-v0
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - name: Download Jest Coverage
+              uses: actions/download-artifact@v2
+              with:
+                  name: jest-coverage
+                  # path to decompress the artifact into
+                  path: ./
+
+            - name: Download Cypress Coverage
+              uses: actions/download-artifact@v2
+              with:
+                  name: cypress-coverage
+                  # path to decompress the artifact into
+                  path: ./
+
             - name: Upload Coverage
               uses: codecov/codecov-action@v3
               with:
                   token: ${{ secrets.CODECOV_TOKEN }}
-                  files: ./coverage/coverage-final.json
+                  files: ./coverage-final.json,./out.json
```

Comment:
This represents the downloaded artifacts, right?  My guess is one is the Jest coverage output and the other is the Cypress coverage output?

Reply:
That's correct.  I'll add a comment.